connectors/es/sink.py

```diff
@@ -824,8 +839,15 @@ def ingestion_stats(self):
             )
         return stats
 
-    def fetch_error(self):
-        return None if self._extractor is None else self._extractor.fetch_error
+    def get_error(self):
+        return (
+            None
+            if self._extractor is None
+            else self._extractor.extractor_error
+            or self._extractor_error
+            or self._sink.sink_error
+            or self._sink_error
```

Comment:
Alternatively, we could try to union multiple errors if more than one of this is non-None?

Reply:
We'll leave that idea for later.